internal/gamepad/.errcheck_excludes_windows

```diff
@@ -0,0 +1 @@
+syscall.Syscall
```

Comment:
Fix this

Reply:
just did :) i got all the excludes files